libs/navigation/api/src/lib/navigation.model.ts

```diff
@@ -33,7 +34,7 @@ export class Navigation {
   @Field()
   key!: string
 
-  @Field(() => [BaseNavigationLink])
+  @FieldProtected(() => [BaseNavigationLink], {permissions: [CanGetNavigations]})
```

Comment:
Should probably always be returned :D

Reply:
Yes indeed, just added this to illustrate.